.github/workflows/ftp-poller-cd.yml

```diff
@@ -0,0 +1,60 @@
+name: FTP Poller CD
+
+on:
+  push:
+    branches:
+      - development
+      - master
+    paths:
+      - "jobs/ftp-poller/**"
+  workflow_dispatch:
+    inputs:
+      environement:
+        description: "Environment (dev/test)"
+        required: true
+        default: "dev"
+
+jobs:
+  ftp-poller-cd:
+    runs-on: ubuntu-latest
+    if: github.repository == 'bcgov/sbc-pay'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set env by input
+        run: |
+          echo "::set-env name=TAG_NAME::${{ github.event.inputs.environement }}"
+
+      - name: Set env by master branch
+        if: env.TAG_NAME == '' && github.ref == 'refs/heads/master'
+        run: |
+          echo "::set-env name=TAG_NAME::test"
+
+      - name: Set env by development branch
+        if: env.TAG_NAME == '' && github.ref == 'refs/heads/development'
+        run: |
+          echo "::set-env name=TAG_NAME::dev"
+
+      - uses: pwei1018/bcrs-cd-action@latest
+        with:
+          WORKING_DIRECTORY: "./jobs/ftp-poller"
+          APP_NAME: "ftp-poller"
+          VAULTS: '[{ \"vault\": \"shared\", \"application\": [\"keycloak\", \"nats\"]}, {\"vault\": \"relationship\",\"application\": [\"postgres-pay\", \"pay-api\", \"sentry\", \"jwt\", \"FTP-poller\"]}]'
```

Comment:
minio can be added here from shared vault, and ftp-poller seems to be defined in lower case in 1password. And may be add payment-reconciliations as well (for Q sub) ?